client/GameData/RealmSynergy.cs

```diff
@@ -81,6 +82,7 @@ static RealmSynergy()
                 new SynergyValue("XII", 112001, Value.FF12),
                 new SynergyValue("XIII", 113001, Value.FF13),
                 new SynergyValue("XIV", 114001, Value.FF14),
+                new SynergyValue("Tactics", 150001, Value.FFT),

```

Comment:
I seem to recall you saying something about it not displaying tactics correctly somewhere - how sure are you that the SynergyValue has completely changed from the pattern and is now 150001 instead of the projected 115001?